lib/Doctrine/DBAL/Driver/PDOStatement.php

```diff
@@ -27,6 +29,29 @@
  */
 class PDOStatement extends \PDOStatement implements Statement
```

Comment:
In this case we keep the dependency?

Reply:
Why? This class will be only loaded if a PDO driver is used in runtime. Currently, all tests pass w/o the PDO extension loaded.